tests/integration/mci/test_backwards_navigation_after_submission.py

```diff
@@ -1,11 +1,13 @@
 from tests.integration.mci import mci_test_urls
-from .test_happy_path import TestHappyPath
+from tests.integration.integration_test_case import IntegrationTestCase
 
+from .test_happy_path import HappyPathHelperMixin
 
-class TestBackwardsNavigationAfterSubmission(TestHappyPath):
+
+class TestBackwardsNavigationAfterSubmission(IntegrationTestCase, HappyPathHelperMixin):
```

Comment:
I just wanted to signpost here that the new mixin class is there because previously, when this inherited from `TestHappyPath`, it meant that all the test cases from `TestHappyPath` were also executed here.

Reply:
Is this similar to this? https://github.com/ONSdigital/eq-survey-runner/blob/master/tests/app/app_context_test_case.py#L39

Reply:
Yes, except that one must be executed a lot more than the one I've fixed

Reply:
If it's okay with you, I'll put through a separate PR which addresses that. Of course it's being collected because pytest is looking for everything starting with `test_`, so I'll just rename that